### ISSUE TITLE
Inject real accounts to DataBundle via java instead of via json

### ISSUE THREAD

**Issue by wkurniawan07** [TEAMMATES/repo#4647]
The idea came out in #4586 (quite long ago) but somehow it got left out.

**Comment by damithc** [comment]
I don't remember the rationale. Can you summarize it here for easier reference?

**Comment by wkurniawan07** [comment]
For easy reference:
1. There are only very few real accounts used so it's a great redundancy if we inject to every single JSON file used.
2. Better organization as we only need to look at the Java file if we want to know whether real accounts are used, and we can even add reasons as to why certain accounts are used where necessary.

**Comment by damithc** [comment]
Got it. Fair enough.
